pkg/master/master.go

```diff
@@ -154,7 +154,6 @@ type completedConfig struct {
 
 // Complete fills in any fields not set that are required to have valid data. It's mutating the receiver.
 func (c *Config) Complete() completedConfig {
-	c.GenericConfig.Complete()
```

Comment:
I think this is still logically required and we'll be working towards eliminating the other call sites as we get certificates more fully baked.  In fact, I think @sttts has already been working towards eliminating the early calls.